### DISCUSSION TITLE
LottieFile component

### DISCUSSION THREAD

**Discussion by willi-piske** [Budibase/budibase#5277] [Ideas]
Hello,

It'd be awesome to have a component that would allow us to place a [lottie](https://lottiefiles.com/) file in the view.
The component's settings tab could have the same options as the tool Lottiefiles.com provides to generate code for the web pages.

![LottieFiles web player config](https://user-images.githubusercontent.com/90471135/162107469-8e3854e2-94b7-423d-a159-da757c0135f4.png)

I think it would be a pretty straightforward way to implement a solution to provide some animation to budibase layouts which, speaking as a designer, allow us to take the user experience to the next level.

Thanks!

**Comment by melohagan** [comment]
Hello @willi-piske 

This seems like a good candidate for a custom component. The ability to build custom components is road mapped for the end of the year: https://github.com/Budibase/budibase/projects/10

**Reply by willi-piske** [reply]
Awesome! Can't wait! :heart_eyes:
